qiskit/compiler/transpiler.py

```diff
@@ -66,6 +66,7 @@ def transpile(  # pylint: disable=too-many-return-statements
     optimization_method: Optional[str] = None,
     ignore_backend_supplied_default_methods: bool = False,
     num_processes: Optional[int] = None,
+    qubits_initially_zero: bool = True,
```

Comment:
nice idea, or maybe `qubits_reset` if we want to avoid the "zero" 🤔

Reply:
Oh yes, I like that suggestion. It is clear and avoids any confusion with the term "zero"  :)

Reply:
FWIW, personally I like the current name `qubits_initially_zero` a bit more than `qubits_reset` or `qubits_zeroed` since it's more clear what it means.